avaframe/log2Report/generateCompareReportCfg.ini

```diff
@@ -2,6 +2,9 @@
 ## Copy to  local_outQuickPlotCfg.ini and set you parameters
 # This file is part of Avaframe.
 
+[GENERAL]
+# limit to produce warning for difference in peak fields
+diffLim = 0.00001
```

Comment:
Should we maybe use percentage instead of a fixed value?

Reply:
Good idea! What percentage of the min difference would be suitable?

Reply:
Mh, that's up for discussion. How about we start with 0.01 % ?

Reply:
fine!